dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js

```diff
@@ -578,18 +592,16 @@ var clientInputDialog = (function() {
             "       <legend class='tool_modal_body_field_label'>" + fldName + "</legend>" +
             "       <div class='tool_modal_body_field_helper_text'>" + fldDescription + "</div>" +
             "       <div class='tool_modal_radio_button_horizontal_wrapper'>" +
-            "           <input id='" + fldId + "-true' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='true'";
-        field += fldDefault === 'true' ? " data-default='true'" : " data-default='false'";
-        field +=
-            "></input>" +
             "           <label for='" + fldId + "-true' class='tool_modal_radio_button_label'>" +
+            "               <input id='" + fldId + "-true' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='true'";
+        field += fldDefault === 'true' ? " data-default='true'" : " data-default='false'";
+        field += ">" +
```

Comment:
</input>?

Reply:
No...not in HTML...from w3schools...

<img width="648" alt="image" src="https://user-images.githubusercontent.com/29490139/70814754-c32a1580-1d91-11ea-90eb-f21991a666fa.png">